bash/solutii/Alex_Tetelea/compare.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+function rec {
+    if [[ "$(diff -r "$1" "$2" | grep -c "Only")" -eq 0 ]]; then
```

Comment:
Voi sti pe viitor, am gasit ceva http://stackoverflow.com/questions/6971284/what-are-the-error-exit-values-for-diff